apps/src/blockly/addons/cdoKeyboardNavigation.ts

```diff
@@ -15,6 +15,11 @@ export function initializeKeyboardNavigation(
 
   createShortcutsModalContainer();
   Blockly.KeyboardNavigation = new KeyboardNavigation(workspace);
+  patchShortcuts({
+    keyboard_nav_copy: 67, // Ctrl+C
```

Comment:
We can use `Blockly.utils.KeyCodes.C` (or similar) for these numbers which would probably make the comments unnecessary too.